compute_endpoint/globus_compute_endpoint/engines/base.py

```diff
@@ -21,11 +21,15 @@
 )
 
 logger = logging.getLogger(__name__)
+_REPORTING_PERIOD = 30.0
 
 
 class ReportingThread:
     def __init__(
-        self, target: t.Callable, args: t.List, reporting_period: float = 30.0
+        self,
+        target: t.Callable,
+        args: t.List,
+        reporting_period: float = _REPORTING_PERIOD,
```

Comment:
Any reason to not set the default to `30.0` and drop the `_REPORTING_PERIOD` variable?

Reply:
Made sense the way I was testing initially. Let me remove this.